com.kristianduske.TrenchBroom.appdata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
The AppData file lacks some data, such as `content_rating` tag (OARS).
https://github.com/flathub/flathub/wiki/AppData-Guidelines
https://hughsie.github.io/oars/generate.html

Reply:
Ok, I answered the questions according to best of my knowledge and got `<content_rating type="oars-1.1" />` as rating. I added it to the appdata.xml. I also did the appstream-util upgrade. It recommended adding donation, translation or help page links which I think this project doesn't have. All documentation ships with the program which is the reason why Pandoc is needed for compilation. But I did add author's email to the file based on the recommendation by this utility.

Should I manually add UserDocs kudo or not?

Reply:
@Sinihopea 

> What does this mean in practical sense? The TB guys need to put the AppData.xml available in their releases?

You should make a pull request in upstream repo about the changes you made here in the AppData file.